OptimizelySDKShared/OptimizelySDKShared/OPTLYClient.m

```diff
@@ -127,6 +127,19 @@ - (BOOL)setForcedVariation:(nonnull NSString *)experimentKey
     }
 }
 
+#pragma mark Forced Variation Methods
+
+- (BOOL)isFeatureEnabled:(nullable NSString *)featureKey userId:(nullable NSString *)userId attributes:(nullable NSDictionary<NSString *,NSString *> *)attributes {
+    if (self.optimizely == nil) {
+        [self.logger logMessage:OPTLYLoggerMessagesClientDummyOptimizelyError
+                      withLevel:OptimizelyLogLevelError];
+        return nil;
+    }
```

Comment:
Compiler warning spotted compiling OptimizelySDKIOSUniversal
/Users/kroach/oakbani/OptimizelySDKShared/OptimizelySDKShared/OPTLYClient.m:136:16: warning: incompatible pointer to integer conversion returning 'void *' from a function with result type 'BOOL' (aka 'signed char') [-Wint-conversion]
        return nil;
               ^~~